irods/session.py

```diff
@@ -203,3 +204,25 @@ def get_connection_refresh_time(self, **kwargs):
                 connection_refresh_time = -1
 
         return connection_refresh_time
+
+    def get_connection_refresh_time(self, **kwargs):
```

Comment:
First, check is connection refresh time is passed to iRODSSession as an input parameter. If not, check the irods env file.